legacy/src/api/bank-accounts/funds-transfers.md

```diff
@@ -165,6 +165,20 @@ A funds transfer represents either a top up to or a withdrawal from a Centrapay
 ]
 {% endjson %}
 
+## Abort Topup
+
+{% reqspec %}
+  POST '/api/funds-transfers/{topupId}/abort'
+  auth 'api-key'
+  path_param 'topupId', '5thg2RPBZEfYTPJdQ63Cre'
+{% endreqspec %}
+
+{% h4 Example response payload %}
+
+{% json %}
+{}
+{% endjson %}
+
```

Comment:
We should add some error codes here for aborting a withdrawal, or aborting a topup in `applied` or `done` state.